app/questionnaire/questionnaire_schema.py

```diff
@@ -603,6 +602,24 @@ def has_address_lookup_answer(question: Mapping) -> bool:
             if answer["type"] == "Address" and "lookup_options" in answer
         )
 
+    def _get_values_for_key(
```

Comment:
This was only being used in questionnaire_schema and was causing a dependancy loop in schema_utils when RuleEvaluator was added. I have also tweaked it as ignore_keys was not been passed on recursion, which it should have been